packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -494,6 +494,14 @@ export class Vector2 {
         return new Vector2(1, 1);
     }
 
+	/**
+	 * Returns a new Vector2 with random values

```

Comment:
I think the doc should explicitely state that the random values are between 0 and 1.

Reply:
Updated - Now accepts (and documents) a min/max